UpsertedProfileOrchestrator/__tests__/handler.test.ts

```diff
@@ -92,32 +88,19 @@ describe("UpsertedProfileOrchestratorV2", () => {
   it.each([
     {
       betaList: [],
-      ff: FeatureFlagEnum.NONE,
-      expectedOrchestrator: "EmailValidationProcessOrchestrator"
-    },
-    {
-      betaList: [aFiscalCode],
-      ff: FeatureFlagEnum.NONE,
-      expectedOrchestrator: "EmailValidationProcessOrchestrator"
+      expectedOrchestrator: "EmailValidationWithTemplateProcessOrchestrator"
     },
     {
       betaList: [aFiscalCode],
-      ff: FeatureFlagEnum.BETA,
       expectedOrchestrator: "EmailValidationWithTemplateProcessOrchestrator"
     },
     {
       betaList: [anotherFiscalCode],
-      ff: FeatureFlagEnum.BETA,
-      expectedOrchestrator: "EmailValidationProcessOrchestrator"
-    },
-    {
-      betaList: [],
-      ff: FeatureFlagEnum.ALL,
       expectedOrchestrator: "EmailValidationWithTemplateProcessOrchestrator"
     }
   ])(
-    "GIVEN an orchestrator with FF: $ff and beta_list: $betaList WHEN the orchestrator start THEN should start the activities with the right inputs",
-    async ({ betaList, ff, expectedOrchestrator }) => {
+    "GIVEN an orchestrator with beta_list: $betaList WHEN the orchestrator start THEN should start the activities with the right inputs",
+    async ({ betaList, expectedOrchestrator }) => {
```

Comment:
Indeed, it seems that refactoring this test makes it irrelevant since it's already covered in other tests.
Solving in [bfc3327](https://github.com/pagopa/io-functions-app/pull/319/commits/bfc3327011770ebf41c4ded7653ebe7da89f53f0).